worker/src/test/java/gov/cms/ab2d/worker/config/AutoScalingServiceTest.java

```diff
@@ -97,7 +97,7 @@ void autoScalingKicksInAndResizes() throws InterruptedException {
         // Last metric taken should always be MAX_POOL_SIZE
         assertThat(new ArrayDeque<>(metrics).getLast(), equalTo(MAX_POOL_SIZE));
         // Some intermediate sizes should be in between, at least 3.
-        assertThat(metrics.size(), greaterThanOrEqualTo(4));
+        assertThat(metrics.size(), greaterThanOrEqualTo(3));
```

Comment:
The comment above is a bit confusing, I agree. 3 intermediate metrics plus one final is 4 total; so we should keep `4`. Let's update the comment instead.